### DISCUSSION TITLE
Reference architecture wizard fails downloading template due to wrong branch ref

### DISCUSSION THREAD

**Discussion by kmox83** [gruntwork-io/knowledge-base#543] [Help] [answered]
Hi,

the wizard of the gruntwork CLI fails while attempting to download `git@github.com:gruntwork-io/gruntwork.git//template?ref=master` because the repository actually have `main` and not `master`

`[boilerplate] 2022/08/24 10:00:00 Downloading templates from git@github.com:gruntwork-io/gruntwork.git//template?ref=master to /var/folders/v3/j7r1gv5524gbs4dc6q6g7ywr0000gn/T/boilerplate-cache2271544320
[boilerplate] 2022/08/24 10:00:04 Cleaning up working directory.
ERROR: error downloading 'ssh://git@github.com/gruntwork-io/gruntwork.git?ref=master': /opt/homebrew/bin/git exited with 1: error: pathspec 'master' did not match any file(s) known to git`

How can we proceed?

---

<ins datetime="2022-08-24T09:17:24Z">
  <p><a href="https://support.gruntwork.io/hc/requests/109163">Tracked in ticket #109163</a></p>
</ins>

**Comment by eak12913** [comment, marked as answer]
Hey @kmox83, thanks so much for the heads up. Our team's looking at this now and will update this post once we have a good answer for you.

**Reply by kmox83** [reply]
Hi, I have sent a PR to fix it, as a workaround I used a local url rewrite.

**Reply by kmox83** [reply]
https://github.com/gruntwork-io/gruntwork/pull/141

**Reply by rhoboat** [reply]
We have fixed this in release [0.3.10](https://github.com/gruntwork-io/gruntwork/releases/tag/v0.3.10).
